config/puma.rb

```diff
@@ -11,5 +11,7 @@
 on_worker_boot do
   # Worker specific setup for Rails 4.1+
   # See: https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#on-worker-boot
-  ActiveRecord::Base.establish_connection
+  ActiveSupport.on_load(:active_record) do
```

Comment:
You can probably revert this change too -- I switched us back to `unicorn` because puma was giving me some unrelated guff.